src/test/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfigTests.java

```diff
@@ -37,5 +38,8 @@ public void checkConfigurationManually() {
         assertTrue(artifactManagerFactory.getProvider() instanceof S3BlobStore);
         assertEquals(((S3BlobStore)artifactManagerFactory.getProvider()).getRegion(), CONTAINER_REGION);
         LOGGER.info(artifactManagerFactory.getProvider().toString());
+
+        //check configuration page submit
+        j.configRoundtrip();
```

Comment:
…but then you have to actually check the result, or you are still not testing anything of any value.

Reply:
it is void, but I can remake the checks that are made before save it

```
    /**
     * Hits the Hudson system configuration and submits without any modification.
     */
    public void configRoundtrip() throws Exception {
        submit(createWebClient().goTo("configure").getFormByName("config"));
    }
```

Reply:
No no, use `JenkinsRule.configRoundtrip`, but afterwards obtain the settings and check that modifications are honored.

For any kind of subtle configuration, you can do stronger tests that persistence works and so on, as seen in the example test case in the global configuration archetype. In this case, the settings _should_ be so simple that no test cases are really needed—mistakes can be easily seen from code inspection.